packages/bot/src/scratch/blocks/Binary/After Purchase/read_details.js

```diff
@@ -16,10 +16,9 @@ Blockly.Blocks.read_details = {
                 },
             ],
             output         : null,
-            outputShape    : Blockly.OUTPUT_SHAPE_ROUND,
```

Comment:
This needs to be reverted because the block glitches if it's not a round shape. You can add a `TODO` explaining that it should be addressed.

```suggestion
            outputShape    : Blockly.OUTPUT_SHAPE_ROUND, // TODO: Investigate why block glitches in square shape.
```